wallet/src/types.rs

```diff
@@ -72,6 +74,7 @@ impl Default for WalletConfig {
 			chain_type: Some(ChainTypes::Floonet),
```

Comment:
Unrelated to your changes but shouldn't this be `ChainTypes::Mainnet`?

Reply:
Looks like it should, yes.